### PR TITLE
MOTECH-2660: Loading state for motech-list

### PR DESCRIPTION
**Pull request by 0nat**
Fixed searching by "search" field

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by sebbrudzinski** [comment]
test this please

File: src/email/logs.contoller.js [hunk 1]
```diff
@@ -40,7 +40,9 @@
 
         var searchUpdateTimeout;
         function updateSearch(searchData){
-            LoadingModal.open();
+            setTimeout(function() {
```
Comment (sebbrudzinski) [inline src/email/logs.contoller.js]: I think both loading modal and the search should use the same timeout. Any reason this is split?

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**